### PR TITLE
[Bug] Disable esync and fsync for windows

### PR DESCRIPTION
**Pull request by Nocccer**
Not sure if this is true but it seems fsync and esync is not available for windows. Prove me wrong.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by arielj — COMMENTED** [review]
I commented on the commit but it makes more sense to comment here on the PR

we are already only showing this for linux non native games only https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/src/frontend/screens/Settings/sections/GamesSettings/index.tsx#L120

I do think it makes sense to move the code here but we should remove conditional in the other places then

File: src/frontend/screens/Settings/components/EnableEsync.tsx [hunk 1]
```diff
@@ -4,12 +4,17 @@ import { ToggleSwitch } from 'frontend/components/UI'
...
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faCircleInfo } from '@fortawesome/free-solid-svg-icons'
+import { isWindows } from 'backend/constants'
 
 const EnableEsync = () => {
   const { t } = useTranslation()
 
   const [enableEsync, setEnableEsync] = useSetting('enableEsync', false)
 
+  if (isWindows) {
```
Comment (arielj) [inline src/frontend/screens/Settings/components/EnableEsync.tsx:14]: in the other code in the GameSettings we only show this config if linux and non native, we should do the same check here (like we do for fsync)

File: src/frontend/screens/Settings/components/EnableFsync.tsx [hunk 2]
```diff
@@ -14,7 +15,7 @@ const EnableFsync = () => {
   const isLinux = platform === 'linux'
   const [enableFsync, setEnableFsync] = useSetting('enableFsync', false)
 
-  if (!isLinux || isLinuxNative) {
+  if (!isLinux || isLinuxNative || isWindows) {
```
Comment (arielj) [inline src/frontend/screens/Settings/components/EnableFsync.tsx:18]: `!isLinux` already covers the os being windows or mac, no need to for extra conditional here

**Comment by Nocccer** [comment]
Ok is already checked in Gamesettings as @arielj pointed out.

**Pull request closed without merging**